<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpreadSheetDisplayEditor</class>
 <widget class="QWidget" name="SpreadSheetDisplayEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="3" column="0" colspan="3">
    <widget class="QTreeWidget" name="CompositeTree">
     <column>
      <property name="text">
       <string>Structure (num. of selected points, cells)</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="ProcessIDLabel">
     <property name="text">
      <string>Process ID: (Range 0 - 0)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="AttributeMode">
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
     <item>
      <property name="text">
       <string>Points</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cells</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Field Data</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vertices</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Edges</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rows</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="ProcessID">
     <property name="alignment">
      <set>Qt::AlignRight</set>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="ViewData">
     <property name="text">
      <string>Show data</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select Attributes to Show</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ViewData</sender>
   <signal>toggled(bool)</signal>
   <receiver>AttributeMode</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
